main.go

```diff
@@ -20,8 +20,7 @@ func run(args []string, exit func(int)) {
 		`A vault for securely storing and accessing AWS credentials in development environments.`,
 	)
 
-	app.ErrorWriter(os.Stderr)
-	app.Writer(os.Stdout)
```

Comment:
This was overriding the previously set value and is deprecated: https://github.com/alecthomas/kingpin/blob/b6657d9477a694e178dcba813954013bfd2f0346/app.go#L133